v2/cli/init_command.go

```diff
@@ -0,0 +1,248 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+type InitFields struct {
+	ID       string
```

Comment:
I would suggest changing this to `ProjectID` to make it a bit more clear what this ID is for.